pkg/xray/util_test.go

```diff
@@ -111,3 +111,13 @@ func DeleteProject(t *testing.T, projectKey string) {
 		t.Fatal(err)
 	}
 }
+
+func generateListOfNames(prefix string, number int) string {
+	var CVEs []string
+	n := 0
+	for n < number {
+		CVEs = append(CVEs, fmt.Sprintf("\"%s%d\",", prefix, test.RandomInt()))
+		n++
+	}
+	return fmt.Sprintf("%s", CVEs)
```

Comment:
Why not:

```go
    var CVEs []string
	n := 0
	for n < number {
		CVEs = append(CVEs, fmt.Sprintf("\"%s%d\"", prefix, test.RandomInt()))
		n++
	}
	return strings.Join(CVEs, ",")
```

Reply:
Because then it doesn't do what I need to get - a list as a string. Like this, it will error out with 
```Error: Unexpected comma after argument```

Reply:
`strings.Join()` does exactly that. Join a slice of strings into a single string, separately by the separation character.

Reply:
https://pkg.go.dev/strings#Join